src/webgl/p5.RenderBuffer.js

```diff
@@ -27,45 +49,77 @@ p5.RenderBuffer = class {
       model = geometry;
     }
 
+    let geometryData = geometry;
+    if (this._namespace) {
+      for (const prefix of this._namespace) {
+        if (!geometryData[prefix]) {
+          geometryData[prefix] = {};
+        }
+        geometryData = geometryData[prefix];
+      }
+    }
+    let modelData = model;
+    if (this._namespace) {
+      for (const prefix of this._namespace) {
+        if (!modelData[prefix]) {
+          modelData[prefix] = {};
+        }
+        modelData = modelData[prefix];
+      }
+    }
+
     // loop through each of the buffer definitions
     const attr = attributes[this.attr];
     if (!attr) {
       return;
     }
 
     // check if the model has the appropriate source array
-    let buffer = geometry[this.dst];
-    const src = model[this.src];
+    let buffer = geometryData[this.dst];
```

Comment:
does buffer ever get changed? Could it be const as well? (I know you didn't write this initially but might as well update while we're here)